packages/mobile/src/escrow/ReclaimPaymentConfirmationScreen.tsx

```diff
@@ -94,28 +103,41 @@ class ReclaimPaymentConfirmationScreen extends React.Component<Props> {
     return <ReviewHeader title={title} />
   }
 
+  renderFooter = () => {
+    return this.props.isReclaiming ? (
+      <ActivityIndicator size="large" color={colors.celoGreen} />
+    ) : null
+  }
+
   render() {
-    const { t, fee } = this.props
+    const { t, isReclaiming, appConnected } = this.props
     const payment = this.getReclaimPaymentInput()
+    const convertedAmount = divideByWei(payment.amount.toString())
```

Comment:
Why do we need a toString() here?
It'd be more clever to make divideByWei accept other types, including null values :)

Reply:
it's bc `payment.amount` is a  `BigNumber` not bc its null 
`divideByWei` can't take BigNums

Now that I think if it, `BigNumber.valueOf()` might be better and more explicit than `toString()`. Will change it to that